// @ant-design/pro-layout/lib/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "DefaultFooter", {
  enumerable: true,
  get: function get() {
    return _Footer.DefaultFooter;
  }
});
Object.defineProperty(exports, "DefaultHeader", {
  enumerable: true,
  get: function get() {
    return _Header.DefaultHeader;
  }
});
Object.defineProperty(exports, "FooterToolbar", {
  enumerable: true,
  get: function get() {
    return _FooterToolbar.FooterToolbar;
  }
});
Object.defineProperty(exports, "GridContent", {
  enumerable: true,
  get: function get() {
    return _GridContent.GridContent;
  }
});
Object.defineProperty(exports, "PageContainer", {
  enumerable: true,
  get: function get() {
    return _PageContainer.PageContainer;
  }
});
Object.defineProperty(exports, "PageHeader", {
  enumerable: true,
  get: function get() {
    return _PageHeader.PageHeader;
  }
});
Object.defineProperty(exports, "PageLoading", {
  enumerable: true,
  get: function get() {
    return _PageLoading.PageLoading;
  }
});
Object.defineProperty(exports, "ProBreadcrumb", {
  enumerable: true,
  get: function get() {
    return _PageContainer.ProBreadcrumb;
  }
});
Object.defineProperty(exports, "ProLayout", {
  enumerable: true,
  get: function get() {
    return _ProLayout.ProLayout;
  }
});
Object.defineProperty(exports, "ProPageHeader", {
  enumerable: true,
  get: function get() {
    return _PageContainer.ProPageHeader;
  }
});
Object.defineProperty(exports, "RouteContext", {
  enumerable: true,
  get: function get() {
    return _RouteContext.RouteContext;
  }
});
Object.defineProperty(exports, "SettingDrawer", {
  enumerable: true,
  get: function get() {
    return _SettingDrawer.SettingDrawer;
  }
});
Object.defineProperty(exports, "TopNavHeader", {
  enumerable: true,
  get: function get() {
    return _TopNavHeader.TopNavHeader;
  }
});
Object.defineProperty(exports, "WaterMark", {
  enumerable: true,
  get: function get() {
    return _WaterMark.WaterMark;
  }
});
exports.default = void 0;
Object.defineProperty(exports, "getMenuData", {
  enumerable: true,
  get: function get() {
    return _getMenuData.getMenuData;
  }
});
Object.defineProperty(exports, "getPageTitle", {
  enumerable: true,
  get: function get() {
    return _getPageTitle.getPageTitle;
  }
});
require("antd/es/anchor/style");
require("antd/es/avatar/style");
require("antd/es/breadcrumb/style");
require("antd/es/divider/style");
require("antd/es/drawer/style");
require("antd/es/layout/style");
require("antd/es/list/style");
require("antd/es/menu/style");
require("antd/es/message/style");
require("antd/es/popover/style");
require("antd/es/select/style");
require("antd/es/space/style");
require("antd/es/spin/style");
require("antd/es/switch/style");
require("antd/es/tabs/style");
require("antd/es/tooltip/style");
var _FooterToolbar = require("./components/FooterToolbar");
var _GridContent = require("./components/GridContent");
var _PageContainer = require("./components/PageContainer");
var _PageHeader = require("./components/PageHeader");
var _Footer = require("./components/Footer");
var _Header = require("./components/Header");
var _PageLoading = require("./components/PageLoading");
var _SettingDrawer = require("./components/SettingDrawer");
var _TopNavHeader = require("./components/TopNavHeader");
var _WaterMark = require("./components/WaterMark");
var _RouteContext = require("./context/RouteContext");
var _getPageTitle = require("./getPageTitle");
var _ProLayout = require("./ProLayout");
var _getMenuData = require("./utils/getMenuData");
// 兼容代码-----------
//----------------------
var _default = _ProLayout.ProLayout;
exports.default = _default;
